ipinfo/countries.go

```diff
@@ -1,260 +1,285 @@
-	"IM": "Isle of Man",
-	"UG": "Uganda",
-	"TZ": "Tanzania",
-	"MY": "Malaysia",
-	"MX": "Mexico",
-	"IL": "Israel",
-	"FR": "France",
-	"IO": "British Indian Ocean Territory",
-	"SH": "Saint Helena",
-	"FI": "Finland",
-	"FJ": "Fiji",
-	"FK": "Falkland Islands",
-	"FM": "Micronesia",
-	"FO": "Faroe Islands",
-	"NI": "Nicaragua",
-	"NL": "Netherlands",
-	"NO": "Norway",
-	"NA": "Namibia",
-	"VU": "Vanuatu",
-	"NC": "New Caledonia",
-	"NE": "Niger",
-	"NF": "Norfolk Island",
-	"NG": "Nigeria",
-	"NZ": "New Zealand",
-	"NP": "Nepal",
-	"NR": "Nauru",
-	"NU": "Niue",
-	"CK": "Cook Islands",
-	"XK": "Kosovo",
-	"CI": "Ivory Coast",
-	"CH": "Switzerland",
-	"CO": "Colombia",
-	"CN": "China",
-	"CM": "Cameroon",
-	"CL": "Chile",
-	"CC": "Cocos Islands",
-	"CA": "Canada",
-	"CG": "Republic of the Congo",
-	"CF": "Central African Republic",
-	"CD": "Democratic Republic of the Congo",
-	"CZ": "Czech Republic",
-	"CY": "Cyprus",
-	"CX": "Christmas Island",
-	"CR": "Costa Rica",
-	"CW": "Curacao",
-	"CV": "Cape Verde",
-	"CU": "Cuba",
-	"SZ": "Swaziland",
-	"SY": "Syria",
-	"SX": "Sint Maarten",
-	"KG": "Kyrgyzstan",
-	"KE": "Kenya",
-	"SS": "South Sudan",
-	"SR": "Suriname",
-	"KI": "Kiribati",
-	"KH": "Cambodia",
-	"KN": "Saint Kitts and Nevis",
-	"KM": "Comoros",
-	"ST": "Sao Tome and Principe",
-	"SK": "Slovakia",
-	"KR": "South Korea",
-	"SI": "Slovenia",
-	"KP": "North Korea",
-	"KW": "Kuwait",
-	"SN": "Senegal",
-	"SM": "San Marino",
-	"SL": "Sierra Leone",
-	"SC": "Seychelles",
-	"KZ": "Kazakhstan",
-	"KY": "Cayman Islands",
-	"SG": "Singapore",
-	"SE": "Sweden",
-	"SD": "Sudan",
-	"DO": "Dominican Republic",
-	"DM": "Dominica",
-	"DJ": "Djibouti",
-	"DK": "Denmark",
-	"VG": "British Virgin Islands",
-	"DE": "Germany",
-	"YE": "Yemen",
-	"DZ": "Algeria",
-	"US": "United States",
-	"UY": "Uruguay",
-	"YT": "Mayotte",
-	"UM": "United States Minor Outlying Islands",
-	"LB": "Lebanon",
-	"LC": "Saint Lucia",
-	"LA": "Laos",
-	"TV": "Tuvalu",
-	"TW": "Taiwan",
-	"TT": "Trinidad and Tobago",
-	"TR": "Turkey",
-	"LK": "Sri Lanka",
-	"LI": "Liechtenstein",
-	"LV": "Latvia",
-	"TO": "Tonga",
-	"LT": "Lithuania",
-	"LU": "Luxembourg",
-	"LR": "Liberia",
-	"LS": "Lesotho",
-	"TH": "Thailand",
-	"TF": "French Southern Territories",
-	"TG": "Togo",
-	"TD": "Chad",
-	"TC": "Turks and Caicos Islands",
-	"LY": "Libya",
-	"VA": "Vatican",
-	"VC": "Saint Vincent and the Grenadines",
-	"AE": "United Arab Emirates",
-	"AD": "Andorra",
-	"AG": "Antigua and Barbuda",
-	"AF": "Afghanistan",
-	"AI": "Anguilla",
-	"VI": "U.S. Virgin Islands",
-	"IS": "Iceland",
-	"IR": "Iran",
-	"AM": "Armenia",
-	"AL": "Albania",
-	"AO": "Angola",
-	"AQ": "Antarctica",
-	"AS": "American Samoa",
-	"AR": "Argentina",
-	"AU": "Australia",
-	"AT": "Austria",
-	"AW": "Aruba",
-	"IN": "India",
-	"AX": "Aland Islands",
-	"AZ": "Azerbaijan",
-	"IE": "Ireland",
-	"ID": "Indonesia",
-	"UA": "Ukraine",
-	"QA": "Qatar",
-	"MZ": "Mozambique",
+	"BD": `{"name":"Bangladesh","isEU":false}`,
```

Comment:
Wrong approach; we need to make this a structure, namely the one you defined above, and have the fields filled in. The map type will change from `map[string]string` to `map[string]CountryDetails`.